book/enterprise-contract.md

```diff
@@ -0,0 +1,171 @@
+
+### Built-in requirements
+
+The built-in requirements are as follows:
+
+- The container image is signed with a known and trusted key
+- The image has an attestation, also signed with a known and trusted key
+
+### Rule-based requirements
+
+The rule-based requirements are based on the content of the pipeline run
+attestation and are defined using [Rego](https://tekton.dev/docs/chains/), the
+[Open Policy Agent](https://tekton.dev/docs/chains/) query language.
+
+Some examples of rule-based requirements are:
+
+- Tasks used in the pipeline run were defined in known and trusted task bundles
+- A defined set of tests were run during the pipeline build with passing results
+
+The technique of checking for a specific task result from a specific known and
+trusted task definition is a useful way to create a robust policy rule. The
+rego language is flexible and expressive so it's easy to create arbitrary
+policy rules based on anything exposed in pipeline run attestation.
+
+
+Components
+----------
+
+### EC CLI
+
+The ec-cli is a command line utility written in Go. Its primary purpose is to
+perform the EC policy validation, which it does as follows:
+
+- For each image included in the release[^2]
+    - Confirm the image is signed and verify the signature
+    - Confirm the image has a signed and verifiable attestation
+    - For each "policy source group"[^3] defined in the ECP CRD config:
+        - Download all defined policy (rego) sources
+        - Download all defined data sources
+        - Run conftest against the image's attestation using those policies and data
+- Output results in JSON format showing details about failures, warnings or violations produced
+
+The ec-cli also supports other related functions. For more information on
+ec-cli refer to the
+[documentation](https://hacbs-contract.github.io/ec-cli/main/reference.html)
+and the [code](https://github.com/hacbs-contract/ec-cli).
+
+### EC Task Definition
+
+The EC Task Definition defines how the ec-cli command should be run in a
+Tekton task. It handles the task inputs and outputs and calls the ec-cli as
+needed to perform the EC validation.
+
+The task is defined
+[here](https://github.com/hacbs-contract/ec-cli/blob/main/deployments/verify-enterprise-contract.yaml).
+
+### EC Policy CRD
+
+The ECP CRD defines a Kubernetes CR which is used to hold the configuration
+needed for running a specific instance of the Enterprise Contract. This
+includes the public key required to verifying signatures, the list of policy
+and data sources, and any other required configuration.
+
+You can view the source code for the ECP CRD
+[here](https://github.com/hacbs-contract/enterprise-contract-controller) and
+see its documentation [here](https://hacbs-contract.github.io/ecc/main/).
+See also the related
+[API Reference](https://redhat-appstudio.github.io/book/ref/enterprise-contract.html)
+
+### EC Policies
+
+The reference set of policy rules created for the HACBS MVP are defined
```

Comment:
I suppose this is going to be the location for default/reference policies not only for MVP and afterwards too. Let's remove the qualifier "HACBS MVP" only.